cmd/ndm_daemonset/controller/controller.go

```diff
@@ -213,6 +225,32 @@ func (c *Controller) setHostName() error {
 	} else {
 		c.NodeAttributes[HostNameKey] = hostName
 	}
+
+	labelList, err := getLabelList()
```

Comment:
why do we need to call this again. we already have the label list in `controller.LabelList` right?

Reply:
Yes, we don't need but if we are planning to remove from controller struct then we need it here.